app/controllers/ajax_calls_controller.rb

```diff
@@ -66,6 +66,31 @@ def load_metereological
     end
     render json: { result: @result, variable: variable, timestamp: timestamp}, layout: true
   end
+  def load_panel
+    variable = params[:variable]
+    units = params[:units]
+    variable = "created_at" if variable.downcase == "last_update"
+    @result = PanelCondition.last[variable] if !PanelCondition.last.nil?
+    @result = "#{@result.to_i}#{units(variable)}" if units == "true"
+    @result = 'N/A' if @result.blank?
+    case variable
+    when "temp_panel"
+      variable = "panel_temp"
+      timestamp = "Control Room"
+    when "temp_ext"
+      variable = "ext_temp"
+      timestamp = "Control Room"
+    when "radiation"
+      variable = "radiation_panel"
+      timestamp = "Control Room"
+    when "created_at"
+      @result = ElectricalMeasurement.last[variable] if !ElectricalMeasurement.last.nil?
```

Comment:
Why using Electrical Measurement model? You should be using your own model here.